CodeStyle/checkstyle.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <property name="charset" value="UTF-8"/>
+    <property name="severity" value="error"/>
+    <property name="fileExtensions" value="java,"/>
+    <module name="SuppressionCommentFilter">
+        <property name="offCommentFormat" value="CHECKSTYLE:OFF"/>
+        <property name="onCommentFormat" value="CHECKSTYLE:ON"/>
+    </module>
+    <module name="FileTabCharacter"/>
+    <module name="Header">
+ 	 <property name="headerFile" value="${basedir}/licenseheader.txt"/>  
+   </module>
+    <module name="NewlineAtEndOfFile"/>
+    <module name="TreeWalker">
+        <!--FileChecks-->
+        <module name="DiamondOperatorForVariableDefinitionCheck"/>
+        <module name="FileContentsHolder"/>
+        <module name="IllegalTokenText">
+            <property name="tokens" value="STRING_LITERAL, CHAR_LITERAL"/>
+            <property name="format"
+                      value="\\u00(08|09|0(a|A)|0(c|C)|0(d|D)|22|27|5(C|c))|\\(0(10|11|12|14|15|42|47)|134)"/>
+            <property name="message" value="Avoid using corresponding octal or Unicode escape."/>
+        </module>
+        <module name="IllegalInstantiation">
+            <property name="classes" value="java.security.InvalidParameterException"/>
+            <!-- sh.ebx.tools.exceptions.MalformedURLExceptionEbx should always be used -->
```

Comment:
The following IllegalInstantiation can be removed - MalformedURLException & DefaultFacebookClient.